openassessment/assessment/api/staff.py

```diff
@@ -3,11 +3,14 @@
 """
 import logging
 from django.db import DatabaseError, IntegrityError, transaction
+from django.utils.timezone import now
 from dogapi import dog_stats_api
 
+from submissions import api as submissions_api
+
```

Comment:
Just out of curiosity, is there any standard pattern as to how to space out `import` statements? (blank line between or no)? I usually just go with "whatever looks easy to read", but if there's a standard I'd love to start using that.